functions/go/set-labels/transformer/labels.go

```diff
@@ -92,7 +92,8 @@ func (p *LabelTransformer) setLabelsInSpecs(o *fn.KubeObject) error {
 
 // Transform updates the labels in the right path using GVK filter and other configurable fields
 func (p *LabelTransformer) Transform(objects fn.KubeObjects) error {
-	for _, o := range objects {
+	// check if local config
```

Comment:
super nit: can you remove this comment or change it to something like "local config is ...., so it should not have labels added"